tools/swift-demangle/swift-demangle.cpp

```diff
@@ -223,10 +224,18 @@ static void demangle(llvm::raw_ostream &os, llvm::StringRef name,
   DCtx.clear();
 }
 
+std::string correctName(llvm::StringRef name) {
```

Comment:
`static` please.

Reply:
Perhaps it's useful to put this logic in-line. It would avoid a few `std::string` allocations in the common case, and it's not all that bad to repeat yourself if you're just going to build up some `llvm::Twine`